currency_rate_update/models/res_currency_rate_provider.py

```diff
@@ -195,7 +195,7 @@ def _process_rate(self, currency, rate):
 
         Module = self.env["ir.module.module"]
 
-        currency_rate_inverted = Module.sudo().search(
```

Comment:
Why? Previous code was ok

Reply:
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch sudo() is deprecated?

Reply:
sudo(user) is, not sudo(). Is documentation not clear enough?

Reply:
Oh! Actually, I assumed that entire method was deprecated

Reply:
Updated